src/components/imageUpload.tsx

```diff
@@ -0,0 +1,17 @@
+"use client";
+
+import React, { useState } from "react";
+import { FileUploader } from "react-drag-drop-files";
+const fileTypes = ["JPG", "PNG", "JPEG"];
+export default function ImageUpload() {
+  const [file, setFile] = useState<File>();
+
+  const handleChange = (file: File) => {
```

Comment:
this all looks great! instead of having `handleChange` in the component, could you make `handleChange` a prop so the parent component can define the behavior? also make sure to define the prop type as an interface

Reply:
Sure, I will modify the component so that the states and handleChange() will be handled by the parent component.